* The following miscellaneous changes have been made.
  Patch: misc-unseparated.diff

  * Various documentation references have been adjusted for Debian.

    References to /usr/local/... have been changed to
    /usr/... as appropriate, etc.
    
Index: sid/etc/NEWS
===================================================================
--- sid.orig/etc/NEWS
+++ sid/etc/NEWS
@@ -35,6 +35,11 @@
 ** cua.el, cua-mode.el: remove old versions.
 
 
+* Debian specific changes to Emacs
+
+Please see /usr/share/doc/emacs22-common/README.Debian.gz.
+
+
 * Installation Changes in Emacs 22.1
 
 ** You can build Emacs with Gtk+ widgets by specifying `--with-x-toolkit=gtk'
